ReleaseNotes-2.9

```diff
@@ -63,6 +69,14 @@ Bugfixes:
  * Deletion of groups with users was not working
  * Fixed generation of the breadcrumb for the Image Editor (Kiwi)
  * Using ActiveSupport::JSON as encoder everywhere prevent to fail on Notification creation with events that have a very big payload
+ * Changes have been reduced in the Kiwi XML layout when modifications are made via the Kiwi Editor
+ * Request does not include element for change
+ * Kiwi importer drops directories
+ * Fix bsc#1068874 in OBS 2.8
```

Comment:
this is for the 2.9 release? ;)

Please don't mention single bugs, except they had been really a big issue.

And please don't put just the number here, but some description instead.

Reply:
Ops. Removed.